examples/spec/integration/search_attributes_spec.rb

```diff
@@ -47,24 +47,6 @@ def cleanup
     end.to raise_error(Temporal::SearchAttributeAlreadyExistsFailure)
   end
 
-  it 'type change fails' do
```

Comment:
@jeffschoner-stripe, this worked fine in the circleci version and I figure it's not really important functionality to test-- like if temporal fixes it, everyone will be happy--so I just removed the test.

Reply:
I think this is another difference between SQL-based enhanced visibility and Elasticsearch-based visibility. I'm fine removing it for now, as we're mostly testing Temporal server, not temporal-ruby here.